Libplanet.Standalone/Hosting/LibplanetNodeService.cs

```diff
@@ -136,7 +136,11 @@ public class LibplanetNodeService<T> : IHostedService, IDisposable
                 listenPort: Properties.Port,
                 iceServers: shuffledIceServers,
                 workers: Properties.Workers,
-                differentAppProtocolVersionEncountered: Properties.DifferentAppProtocolVersionEncountered
+                differentAppProtocolVersionEncountered: Properties.DifferentAppProtocolVersionEncountered,
+                options: new SwarmOptions
+                {
+                    MaxTimeout = TimeSpan.FromSeconds(10)
```

Comment:
How about changing `BlockHashRecvTimeout` as well?
```suggestion
                    MaxTimeout = TimeSpan.FromSeconds(10),
                    BlockHashRecvTimeout = TimeSpan.FromSeconds(10),
```